src/components/NavBar/NavigationBar.cy.tsx

```diff
@@ -32,37 +41,37 @@ describe("<NavigationBar />", () => {
         cy.viewport(1366, 768);
         cy.mount(RouterWrappedNavBar);
 
-        cy.get("a[href='/clients']").contains("Clients").should("exist");
-        cy.get("a[href='/lists']").contains("Lists").should("exist");
-        cy.get("a[href='/calendar']").contains("Calendar").should("exist");
+        cy.get("a[href='/clients']").contains("CLIENTS").should("exist");
+        cy.get("a[href='/lists']").contains("LISTS").should("exist");
+        cy.get("a[href='/calendar']").contains("CALENDAR").should("exist");
     });
 
     it("Links exist on mobile", () => {
         cy.viewport(360, 760);
         cy.mount(RouterWrappedNavBar);
 
         cy.get("button[aria-label='Mobile Menu Button']").click();
-        cy.get("a[href='/clients']").contains("Clients").should("exist");
-        cy.get("a[href='/lists']").contains("Lists").should("exist");
-        cy.get("a[href='/calendar']").contains("Calendar").should("exist");
+        cy.get("a[href='/clients']").contains("CLIENTS").should("exist");
```

Comment:
I changed the default MuiButton to not capitalize button text. Mainly because our login page green button is not capitalize too and in general it seems more convenient to only capitalize as we want (most of the buttons we use in other pages looks nicer without capitalization - which is consistent to the original designs by others on different components).

However, the nav bar originally is capitalized and it does look nicer capitalized - so I just change the nav bar text to be actually capitalized to retain same effect.

The test however reads the text I fed in as props, so now I have to update the test to make it pass